lib/Parser/Parser.cpp

```diff
@@ -40,8 +40,13 @@ struct Token {
     Int,
     UntypedInt,
     KnownBits,
-    MoreKnownBits,
     NumSignBits,
+    PowerOfTwo,
+    NonNegative,
+    Negative,
+    NonZero,
+    OpenParen,
+    CloseParen,
```

Comment:
you should eliminate all of these kinds of tokens. rather, they can lex as Ident.

Reply:
I mean eliminate: 
￼      KnownBits,
￼ -    MoreKnownBits,
￼      NumSignBits,
￼ +    PowerOfTwo,
￼ +    NonNegative,
￼ +    Negative,
￼ +    NonZero,